packages/components/src/components/ScalarListbox/ScalarListbox.vue

```diff
@@ -58,8 +57,6 @@ const variants = cva({
 <template>
   <Listbox
     v-slot="{ open }"
-    as="div"
```

Comment:
It will render as [`<template>`](https://headlessui.com/v1/vue/listbox#listbox) now